src/components/cursor.js

```diff
@@ -33,60 +38,118 @@ module.exports.Component = registerComponent('cursor', {
   dependencies: ['raycaster'],
 
   schema: {
+    downEvents: {default: []},
```

Comment:
See https://github.com/aframevr/aframe/pull/2678/files#diff-3f66fed14508fae90dc58b66fac99025R36 for usage